src/note/asset_base.rs

```diff
@@ -101,7 +101,7 @@ impl AssetBase {
     /// Generates a ZSA random asset.
     ///
     /// This is only used in tests.
-    pub(crate) fn random(rng: &mut impl RngCore) -> Self {
+    pub(crate) fn random(rng: &mut impl CryptoRngCore) -> Self {
```

Comment:
We don't want a new dependency `k256::schnorr::CryptoRngCore;` unless absolutely needed,

Reply:
I'd used it at a point, but it was no longer needed and I missed removing it. Removed it now, thanks.